### DISCUSSION TITLE
Closing Window with KeyboardInterrupt

### DISCUSSION THREAD

**Discussion by SirNate0** [pyvista/pyvista#5455] [Q&A]
I would like to be able to close my windows using either Ctrl+C in the terminal, or hitting the close button on the window. However, I have only been able to get one of them working at a time, with a possible bug encountered for one of the options:

```py
import pyvista

plotter = pyvista.Plotter(window_size=(800, 600))
plotter.show_axes() # so we can see something in the window

if 0:
    # With this, KeyboardInterrupt is only sent after I close the window
    plotter.show()
else:
    # With this, KeyboardInterrupt works to close the script, but closing the window 
    # results in a loop of window closing and window opening until the 120 frames are finished.
    plotter.show(interactive_update=True)
    for i in range(120):
        plotter.update()
```

**Comment by tuu4208** [comment]
Linux: 
- cancel command(ctrl+c)
- force delete process (ctrl+/)

You can try ctrl+/ instead.

**Reply by SirNate0** [reply]
I am pretty sure you mean Ctrl+\ which sends the QUIT signal (since Ctrl+/ did nothing). Since this seems to [cause a core dump by default](https://stackoverflow.com/a/19248775), I don't think it is a great workaround.
